### ISSUE TITLE
[enterprise-4.7] Issue in file operators/operator_sdk/osdk-leader-election.adoc

### ISSUE THREAD

**Issue by porridge** [openshift/openshift-docs#31509]
### Which section(s) is the issue in?

In the top part, before the first subsection.

### What needs fixing?

The following sentence:

> By default, the Operator SDK enables the Leader-for-life implementation

seems (no longer?) correct. See https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#leader-election which contains the same text but points at the other implementation as the default. I think @SimonBaeumer said that the code confirms that.

**Comment by SimonBaeumer** [comment]
Yes, at least the generation of the go operator created a configuration like this:

```go
	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
		Scheme:                 scheme,
		MetricsBindAddress:     metricsAddr,
		Port:                   9443,
		HealthProbeBindAddress: probeAddr,
		LeaderElection:         enableLeaderElection,
		LeaderElectionID:       "as12ad12.test.example.com",
	})
```

**Comment by bergerhoffer** [comment]
FYI @adellape
